pyomo/core/tests/unit/test_pickle.py

```diff
@@ -227,7 +227,7 @@ def test_pickle_concrete_model_constant_objective(self):
     def test_pickle_concrete_model_objective(self):
         model = ConcreteModel()
         model.x = Var()
-        model.A = Objective(expr=model.x <= 0)
+        model.A = Objective(expr=model.x**2)
```

Comment:
Was this a typo? We actually tested you could set the objective to a relational expression?!

Reply:
Alas, yes.  That test has been there for a long, long time.  I'm pretty sure it was a copy-paste-and-forget-to-update error when the test suite was originally thrown together.